pages/form-detail-dummy.md

```diff
@@ -11,16 +11,114 @@ title: Form detail dummy page
+  </div>
+  <div class="vads-u-margin-y--1">
+    <dt class="vads-u-font-weight--bold vads-u-display--inline">
+      Related to:
+    </dt><dd class="vads-u-display--inline">
+            Health care
+                  </dd>
+  </div>
+    <div>
+      <dt class="vads-u-font-weight--bold vads-u-display--inline">
+        Form last updated:
+      </dt>
+      <dd class="vads-u-display--inline">
+        January 11, 2020
+      </dd>
+    </div>
+</dl>
+  <h2 class="vads-u-margin-top--4" id="when-to-use-this-form">When to use this form</h2>
+  <p>Use this form when you want to test the user experience of form with an invalid PDF.</p>
+  <h3 id="downloadable-pdf">Downloadable PDF</h3>
+  <a href="https://www.va.gov/vaforms/medical/pdf/10-10EZ-fillable.pdf" target="_blank" download="" data-widget-type="find-va-forms-invalid-pdf-alert" data-form-number="VA0927b" rel="noopener ">
+    Download VA Form VA0927b (PDF)
+  </a>
+  <h3 id="online-tool">Online tool</h3>
+  <p>You can apply online instead of filling out and sending us the paper form.</p>
+  <a href="https://www.va.gov/health-care/apply/application/introduction" class="usa-button-primary va-button-primary">
+    Go to the online tool <i role="presentation" aria-hidden="true" class="fa fa-chevron-right vads-u-margin-left--1"> </i>
+  </a>
+  <section>
+    <h2 id="related-forms-and-instructions">Related forms and instructions</h2>
+    <ul class="usa-unstyled-list">
+        <li>
+          <hgroup>
+            <h3 id="va-form-10-10ez-esp">VA Form VA0927b (esp)</h3>
+            <h4 class="vads-u-margin-top--0">
+              <span class="vads-u-visibility--screen-reader">Form name:</span> Instrucciones y Solicitud de Beneficios Medicos
+            </h4>
+          </hgroup>
+          <p>Para que los Veteranos soliciten su afiliación al sistema de atención médica de VA. la información proporcionada en este formulario será utilizada por VA para determinar si usted reúne los requisitos para recibir beneficios médicos.</p>
+          <a href="https://www.va.gov/vaforms/medical/pdf/10-10EZ_Spanish.pdf" target="_blank" download="" data-widget-type="find-va-forms-invalid-pdf-alert" data-form-number="VA0927b" rel="noopener ">
+            Download VA Form VA0927b (esp) (PDF)
+          </a>
+        </li>
+        <li>
+          <hgroup>
+            <h3 id="va-form-10-10ezr">VA Form VA0927b</h3>
+            <h4 class="vads-u-margin-top--0">
+              <span class="vads-u-visibility--screen-reader">Form name:</span> Health Benefits Update Form
+            </h4>
+          </hgroup>
+          <p>Use VA Form VA0927b if you already receive VA health care benefits, and you need to update your personal, insurance, or financial information.</p>
+          <a href="https://www.va.gov/vaforms/medical/pdf/vha-10-10ezrfill.pdf" target="_blank" download="" data-widget-type="find-va-forms-invalid-pdf-alert" data-form-number="VA0927b" rel="noopener ">
```

Comment:
Should we use relative routes for these links? 🙂

Reply:
We could but I think the form URLs are hosted as absolute...or at least that's how I copied them lol.